packages/frontend/components/composed/community/ResearchUnitsList/ResearchUnitsList.styles.tsx

```diff
@@ -0,0 +1,94 @@
+import styled from "styled-components";
+import { pxToRem } from "@wdp/lib/theme/functions";
+import BaseImage from "next/image";
+import { respond, lGrid } from "theme/mixins";
+
+export const Inner = styled.div`
+  ${lGrid()}
+  padding-block-start: ${pxToRem(100)};
+  padding-block-end: ${pxToRem(100)};
+
+  ${respond(
+    `
+      grid-template-columns: 1fr;
+      grid-template-rows: auto;
+      grid-template-areas:
+        "text"
+        "image";
+    `,
+    70
+  )}
+`;
+
+export const TextColumn = styled.div`
+  grid-column: 2 / 6;
+
+  ${respond(
+    `
+      grid-area: text;
+    `,
+    70
+  )}
+`;
+
+export const HeaderBlock = styled.div`
+  padding-block-end: ${pxToRem(12)};
+  border-bottom: 1px solid currentColor;
```

Comment:
There's a root variable `--border-color` that should match design and changes based on the background!